generic-desktop-kde-live-20160731.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "generic-desktop"

```

Comment:
but not Leap - and you match TW branding also on leap. we already have quite some generic-desktop needles ;(